internal/rootcoord/import_manager.go

```diff
@@ -381,25 +382,39 @@ func (m *importManager) checkIndexingDone(ctx context.Context, collID UniqueID,
 	return len(allSegmentIDs) == indexedSegmentCount, nil
 }
 
+func (m *importManager) isRowbased(files []string) (bool, error) {
+	isRowBased := false
```

Comment:
what happens if the files are:

[1.npy, 2.npy, 3.json]

Reply:
also one thought: how about to forget `rowbased in our code as well?

simply consider all json files row-based by default and all numpy files column based?

Reply:
The import_wrapper.fileValidation() will handle it

Reply:
row-based:  all files must be json
column-based: all files must be numpy


importManager.isRowBased() is to know which job is row-based, which job is column-based, so that use different method to create import tasks: in row-based job, each json file convert to a task; in column-based job, all files convert into one task.

Reply:
import_wrapper.fileValidation() will do more validation such as: duplicate file name, check file size, alert redundant files, etc

Reply:
If things are validation in the very beginning. Can't we just look at the extension of the first file to figure out?

Reply:
The importManager.isRowBased() use the first file extension to determine row-based or column-based.

More validation are in importWrapper.fileValidation(), to check file size, we need chunkManager, to check redundant fiels, we need collection schema. The importManager neither know chunkManager not the collection schema.

Reply:
No i mean this implementation of `importManager.isRowBased()` is not correct. If given [1.npy, 2.npy, 3.json], it will return `true` but actually it should return an error.

Reply:
I mean no need to do more validation in importManager.isRowBased(), just let the importWrapper.fileValidation() to do a full validation.
importManager is in RootCoord, shouldn't know detail logic of importWrapper.
For [1.npy, 2.npy, 3.json], the importWrapper.fileValidation() will report error.